package.json

```diff
@@ -10,9 +10,8 @@
   "scripts": {
     "build": "yarn turbo build",
     "clean": "yarn turbo clean && yarn storybook:clean && rimraf node_modules",
-    "test": "jest && yarn test:pkg:aio",
-    "test:pkg:aio": "yarn pkg:aio test && yarn pkg:aio build",
```

Comment:
Oops accidentally deleted this. Does anyone use it (nothing automated calls it)?

Reply:
We won't need it if turbo works